lib/generator/files/exercise_files.rb

```diff
@@ -0,0 +1,83 @@
+require 'erb'
+
+module Generator
+  module Files
+    module GeneratorCases
+      def self.available(track_path)
```

Comment:
I would consider using `extend self` here since this seems to be more of a utility module, that would eliminate the need for self references
```ruby
module GeneratorCases
  extend self
```